### DISCUSSION TITLE
FR: Ability to hide subscriptions & donations

### DISCUSSION THREAD

**Discussion by Mosnar** [craftcms/commerce#2374] [Ideas] [answered]
### Description
I would like to be able to hide these features that we don't use. We have no subscription plans created, so it would be nice to hide the menu item for Subscriptions. We have donations disabled, so the donation product really shouldn't show up in the backend order creation as an option.

### Additional info

- Craft version: 3.4.x
- Craft Commerce version: 3.1.x

**Comment by lukeholder** [comment]
Will be in next release of Commerce. Just merged: https://github.com/craftcms/commerce/pull/2659

**Comment by lukeholder** [comment]
Subscriptions will not show up in the menu if no plans are created: https://github.com/craftcms/commerce/blob/develop/src/Plugin.php#L253-L258

Are you seeing the nav menu item with no plans set up?

**Reply by jonleverrier** [reply]
Subscriptions left nav item doesn't show up if no plan is created @lukeholder. 

But when creating a manual order, the donation line item is always present, despite there being no donation available for purchase.

I've hidden it with CSS for the time being...but this is what I would normally see:
![Screenshot-2022-01-10-at-14 46 28](https://user-images.githubusercontent.com/2373940/148776729-e3de56c5-5bac-4d37-95e7-8fb02252eb51.jpg)

**Reply by lukeholder** [reply]
Right, we fixed the donation showing up issue in my link to #2659.

I was just asking about the Subscription menu item also referenced in the initial question.

**Comment by lukeholder** [comment, marked as answer]
In addition to the previous bug fixes, we have allowed any navigation item to be globally hidden for the next major release in this PR: https://github.com/craftcms/commerce/pull/3914
